### PR TITLE
Add new resource naming article

### LINKED ISSUE #18304: Add new article on how resource files are named in .NET Core
Related to https://github.com/dotnet/docs/issues/16964.

Also, add UseDependentUponConvention property to msbuild-props.md.

### PR DESCRIPTION
**Pull request by gewarren**
Fixes #18304.

[Preview - resource manifest naming](https://review.docs.microsoft.com/en-us/dotnet/core/resources/manifest-file-names?branch=pr-en-us-18307)
[Preview - MSBuild property](https://review.docs.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props?branch=pr-en-us-18307#embeddedresourceusedependentuponconvention)

### CODE REVIEW

**Review by IEvangelist — APPROVED** [review]
Looks good to me :shipit:

File: docs/core/project-sdk/msbuild-props.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ ms.topic: reference
 ---
 # MSBuild properties for .NET Core SDK projects
 
-This page describes MSBuild properties for configuring .NET Core projects. You can specify *metadata* for each property as child elements of the property.
+This page is a reference for the MSBuild properties that you can use to configure .NET Core projects. You can specify *metadata* for each property either as an attribute or a child element of the property.
```
Comment (rainersigwald) [inline docs/core/project-sdk/msbuild-props.md:9]:
```suggestion
This page is a reference for the MSBuild properties that you can use to configure .NET Core projects.
```

Properties can't have metadata, only items.

File: docs/core/resources/manifest-file-names.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+---
+title: How MSBuild generates manifest file names
+description: Describes the factors that influence the name of a resource manifest file name that's generated by MSBuild at compile time.
+ms.date: 05/08/2020
+ms.topic: conceptual
+---
+# How resource manifest files are named
+
+When MSBuild compiles a .NET Core project, XML resource files, which have the *.resx* file extension, are converted into binary *.resources* files. The binary files can be read by the <xref:System.Resources.ResourceManager>. This article describes how MSBuild chooses a name for each *.resources* file.
```
Comment (rainersigwald) [inline docs/core/resources/manifest-file-names.md:9]:
```suggestion
When MSBuild compiles a .NET Core project, XML resource files, which have the *.resx* file extension, are converted into binary *.resources* files. The binary files are embedded into the output of the compiler and can be read by the <xref:System.Resources.ResourceManager>. This article describes how MSBuild chooses a name for each *.resources* file.
```

**Review by rainersigwald — APPROVED** [review]
cc @ghogen

**Comment by gewarren** [comment]
Thanks for reviewing, David and Rainer!

**Comment by BenVillalobos** [comment]
Should the section for `EmbeddedResourceUseDependentUponConvention` link to the more detailed docs page?

**Comment by gewarren** [comment]
> Should the section for `EmbeddedResourceUseDependentUponConvention` link to the more detailed docs page?

Thanks, yes, I added a link.

**Pull request merged**